docs/tasks/administer-cluster/distribute-credentials-secure.md

```diff
@@ -0,0 +1,122 @@
+---
+---
+
+{% capture overview %}
+This page shows how to create a Secret and a Pod that has access to the Secret.
```

Comment:
Should we phrase this more from the user problem it is trying to solve?  e.g. Injecting sensitive data into Pods, such as encryption keys and passwords